### PR TITLE
Pim 5920 dropzones

### PR DESCRIPTION
**Pull request by pierallard**
This PR unify dropzones everywhere on the PIM.

capture du 2016-08-10 14 40 59

capture du 2016-08-10 14 40 53

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | n
| Added Behats                      | n
| Changelog updated                 | n
| Review and 2 GTM                  | TODO
| Micro Demo to the PO (Story only) | TODO
| Migration script                  | n
| Tech Doc                          | n

### CODE REVIEW

File: src/Pim/Bundle/UIBundle/Resources/public/css/form.less [hunk 1]
```diff
@@ -194,12 +194,6 @@ a.accordion-toggle {
   }
 }
 
-#pim_importexport_import_upload .control-group {
```
Comment (SamirBoulil) [inline src/Pim/Bundle/UIBundle/Resources/public/css/form.less]: 👍

File: src/Pim/Bundle/UIBundle/Resources/public/css/form.less [hunk 2]
```diff
@@ -386,45 +376,34 @@ div.btn-group > .has-switch {
...
 
     img {
       display: block;
     }
 
     span.upload-filename {
-      padding: 0 5px;
-      line-height: @uploadZoneHeight - 20;
+      line-height: @baseFontSize;
+      font-weight:normal;
```
Comment (juliensnz) [inline src/Pim/Bundle/UIBundle/Resources/public/css/form.less]: do you really want a comment on every line?

**Comment by juliensnz** [comment]
GTM

File: src/Pim/Bundle/UIBundle/Resources/public/css/form.less [hunk 3]
```diff
@@ -361,23 +355,19 @@ div.btn-group > .has-switch {
   z-index: 900;
 }
 
+.control-group.upload-zone-container {
+  width: 900px;
```
Comment (SamirBoulil) [inline src/Pim/Bundle/UIBundle/Resources/public/css/form.less]: I'm not expert, but it seems wierd to set the with to 900px and then the max-width ? When you set with: 900px that means that the with is, and will forever be, 900px ?

Maybe drop the max-width

**Comment by SamirBoulil** [comment]
Looks good to me ;) GTM

File: src/Pim/Bundle/UIBundle/Resources/public/css/form.less (hunk 2 above)
Comment (pierallard) [inline src/Pim/Bundle/UIBundle/Resources/public/css/form.less]: :trollface:

File: src/Pim/Bundle/UIBundle/Resources/public/css/form.less (hunk 3 above)
Comment (pierallard) [inline src/Pim/Bundle/UIBundle/Resources/public/css/form.less]: In fact, there is a rule for .control-group specifying max-width at 700px. This one just override the previous one.

Comment (SamirBoulil) [inline src/Pim/Bundle/UIBundle/Resources/public/css/form.less]: the witdth directive cancels this max-width doesn't it ?

Comment (pierallard) [inline src/Pim/Bundle/UIBundle/Resources/public/css/form.less]: nope ! Max width is preemptive for width.

**Pull request merged**